documentation/modules/exploit/linux/http/vestacp_exec.md

```diff
@@ -0,0 +1,84 @@
+## Vulnerable Application
+This module exploits command injection vulnerability in v-list-user-backups bash script file. Low privileged authenticated users can execute arbitrary commands under the context of the root user.
+
+An authenticated attacker with a low privileges can inject a payload in the file name starts with dot. During the user backup process, this file name will be evaluated by the v-backup-user bash scripts. As result of that backup process, when an attacker try to list existing backups injected payload will be executed.
+        
+## Vulnerable Application Installation Steps
```

Comment:
This and the title above it should really be a bit more specific to what operating systems it supports. If your installing on Ubuntu 18.03 LTS for example, the tile should reflect this, and state something like "Installing the Vulnerable Application on Ubuntu 18.03 LTS". Its possible that things may break in the future so having accurate logs of what you tested is helpful for these scenarios :)

Also stating "Linux server" here is way to vauge in my opinion given that Linux has many flavors. Update this to be the server you installed it on and its version, so say "Ubuntu 18.04 LTS".

Reply:
Confirmed fixed, good job :)